Pipfile

```diff
@@ -15,6 +15,8 @@ networkx = "*"
 pandas = "*"
 regex = "*"
 xlrd = "*"
+pyyaml = "*"
+types-pyyaml = "*"
```

Comment:
I think `types-pyyaml` could be a dev dependency

Reply:
thanks. it's moved.